packages/bash_completion.rb

```diff
@@ -0,0 +1,26 @@
+require 'package'
+
+class Bash_completion < Package
+  description 'Programmable completion functions for bash'
+  homepage 'https://github.com/scop/bash-completion'
+  version '2.7'
+  source_url 'https://github.com/scop/bash-completion/archive/2.7.tar.gz'
+  source_sha256 'dba2b88c363178622b61258f35d82df64dc8d279359f599e3b93eac0375a416c'
+
+  depends_on 'autoconf'
+  depends_on 'automake'
+
+  def self.build
+    system "autoreconf -i"
+    system "./configure"
+    system "make"
+  end
+
+  def self.install
+    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    puts "Add the floowing to your .bashrc"
+    puts "[[ $PS1 && -f /usr/local/share/bash-completion/bash_completion ]] && \"
```

Comment:
Need one more \ this line, otherwise:
```
$ crew install bash_completion
/usr/local/lib64/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require': /usr/local/lib/crew/packages/bash_completion.rb:23: syntax erro
r, unexpected tSTRING_BEG, expecting keyword_do or '{' or '(' (SyntaxError)
sh-completion/bash_completion"
                              ^
/usr/local/lib/crew/packages/bash_completion.rb:23: unterminated string meets end of file
        from /usr/local/lib64/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from /usr/local/bin/crew:177:in `set_package'
        from /usr/local/bin/crew:193:in `block in search'
        from /usr/local/lib64/ruby/2.4.0/find.rb:49:in `block (2 levels) in find'
        from /usr/local/lib64/ruby/2.4.0/find.rb:48:in `catch'
        from /usr/local/lib64/ruby/2.4.0/find.rb:48:in `block in find'
        from /usr/local/lib64/ruby/2.4.0/find.rb:43:in `each'
        from /usr/local/lib64/ruby/2.4.0/find.rb:43:in `find'
        from /usr/local/bin/crew:192:in `search'
        from /usr/local/bin/crew:792:in `block in install_command'
        from /usr/local/bin/crew:790:in `each'
        from /usr/local/bin/crew:790:in `install_command'
        from /usr/local/bin/crew:840:in `<main>'
```